openedx/features/course_experience/views/course_home.py

```diff
@@ -141,6 +145,11 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         # Get the course tools enabled for this user and course
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        # Grab the course home message fragment
```

Comment:
Is this showing only a single fragment, or could we eventually have multiple messages shown here? If it only shows a single one, what happens if multiple messages are registered? I think I'd make this plural, and allow it to render as many messages as it finds.